docs/TODO.md

```diff
@@ -17,6 +17,7 @@ Things pending or ideas. Don't expect them soon. This is not a roadmap.
 * Weeks are off, currently being calculated as `floor((dayOfMonth - 1) / 7)` which is wrong.
 * Detect if some ES6 or other API used during generation is not available and show an error early.
 * Add other platform parsers like Google Teams, MS Teams, Slack, etc.
+* Handle emoticons? (e.g. :D, :P, etc)
```

Comment:
Not sure about Telegram and WhatsApp, but Discord and Messenger convert emoticons to emojis automatically and the user has to specifically choose to prevent the conversion if they would prefer to use the emoticon, so I don't think this is necessary

Reply:
ChatAnalytics should be agnostic of the platform after parsing, and since text **\*may\*** contain emoticons I think we should handle it eventually. WhatsApp and Telegram don't convert them automatically (at least :)).
Also emoticons can be thing like (╯°□°）╯︵ ┻━┻ where a mapping to a unicode emoji do no exist

Reply:
If we handle it in the future, it will be with a known database of emoticons ofc

Reply:
I thought by "Handle emoticons" you meant to map it to an emoji- so what do you plan on doing with them?

Reply:
Treat them as custom emojis, like :pepe: right now